<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="application/xhtml+xml; charset=UTF-8" />
<link href="stylesheets/guo.css" type="text/css" rel="stylesheet" />
<link href="stylesheets/w3.css" type="text/css" rel="stylesheet" />
<link href="stylesheets/common.css" type="text/css" rel="stylesheet" />
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<title>Dr. Guo's Lab</title>
<script>
      $(function(){
	$("#header-placeholder").load("header.html");
	$("#foot-placeholder").load("foot.html");
}); </script>
</head>
<body>
<div class="main-container">
  <div id="header-placeholder"></div>
  <div class="title">
    <h3>Xuan Guo's Tenure and Promotion Application Dossier</h3>
  </div>
  <div> First of all, thank you very much for considering and evaluating my tenure and promotion application.
    
    I joined the University of North Texas in September 2017. Since then, I have been an Assistant Professor in the Department of Computer Science and Engineering (CSE). I am submitting my application for tenure and promotion. The documents in my dossier are provided on this web page. </div>
  <div class="cv">
    <div class="research-interest">
      <p class="subtitile">UNT tenure and promotion policies</p>
      <ul>
        <li><a href="https://vpaa.unt.edu/sites/default/files/documents/2019/ceng_computer_science_and_engineering_promotion_and_tenure_final.pdf" target="_blank">CSE Department Tenure and Promotion Policies </a></li>
        <li><a href="https://vpaa.unt.edu/file/22282" target="_blank">UNT Faculty Reappointment, Tenure, and Promotion Policies </a></li>
      </ul>
    </div>
    <div class="teaching">
      <p class="subtitile">My tenure and promotion application documents</p>
      <ul>
        <li><a href="1 Curriculum Vitae - Xuan Guo.pdf" target="_blank">Curriculum vitae</a></li>
        <li><a href="2 Self-evaluation Personal Narrative - Xuan Guo.pdf" target="_blank">Self-evaluation personal narrative</a></li>
        <li> <a href="teaching/2022-2018 SPOT.pdf" target="_blank"> Teaching evaluations </a></li>
        <li> Selected publications </li>
        <ol>
          <li><a href="papers/2021 Deep learning for peptide identification from metaproteomics datasets.pdf" target="_blank">Deep learning for peptide identification from metaproteomics datasets</a></li>
          Journal of Proteomics 247 (2021): 104316.
          <li><a href="papers/2021 Predicting psoriasis using routine laboratory tests with random forest.pdf" target="_blank">Predicting psoriasis using routine laboratory tests with random forest</a></li>
          Plos one 16, no. 10 (2021): e0258768.
          <li><a href="papers/2020 TS a powerful truncated test to detect novel disease associated genes using publicly available gWAS summary data.pdf" target="_blank">TS: a powerful truncated test to detect novel disease associated genes using publicly available gWAS summary data</a></li>
          BMC bioinformatics 21, no. 1 (2020): 1-15.
          <li><a href="papers/2020 JS-MA A Jensen-Shannon Divergence Based Method for Mapping Genome-Wide Associations on Multiple Diseases.pdf" target="_blank">JS-MA: A Jensen-Shannon Divergence Based Method for Mapping Genome-Wide Associations on Multiple Diseases</a></li>
          Frontiers in genetics (2020): 1251.
          <li><a href="papers/2019 Genome-resolved proteomic stable isotope probing of soil microbial communities using 13CO2 and 13C-methanol.pdf" target="_blank">Genome-resolved proteomic stable isotope probing of soil microbial communities using 13CO2 and 13C-methanol</a></li>
          Frontiers in microbiology 10 (2019): 2706.
          <li><a href="papers/2019 Powerful statistical method to detect disease-associated genes using publicly available genome-wide association studies summary data.pdf" target="_blank">Powerful statistical method to detect disease-associated genes using publicly available genome-wide association studies summary data</a></li>
          Genetic epidemiology 43, no. 8 (2019): 941-951.
          <li><a href="papers/2018 Community proteogenomics reveals the systemic impact of phosphorus availability on microbial functions in tropical soil.pdf" target="_blank">Community proteogenomics reveals the systemic impact of phosphorus availability on microbial functions in tropical soil</a></li>
          Nature ecology & evolution 2, no. 3 (2018): 499-509.
          <li><a href="papers/2018 Sipros Ensemble improves database searching and filtering for complex metaproteomics.pdf" target="_blank">Sipros ensemble improves database searching and filtering for complex metaproteomics</a></li>
          Bioinformatics 34, no. 5 (2018): 795-802.
          <li><a href="papers/2016 Searching Genome-Wide Multi-Locus Associations for Multiple Diseases Based on Bayesian Inference.pdf" target="_blank">Searching genome-wide multi-locus associations for multiple diseases based on Bayesian inference</a></li>
          IEEE/ACM transactions on computational biology and bioinformatics 14, no. 3 (2016): 600-610.
          <li><a href="papers/2014 Cloud computing for detecting high-order genome-wide epistatic interaction via dynamic clustering.pdf" target="_blank">Cloud computing for detecting high-order genome-wide epistatic interaction via dynamic clustering</a></li>
          BMC bioinformatics 15, no. 1 (2014): 1-16.
        </ol>
      </ul>
    </div>
    <div class="research-interest">
      <p class="subtitile">Contributions to Science</p>
      <ul>
        <li> <b> Metaproteomics Search </b></li>
        Metagenomics and metaproteomics can characterize complex microbial communities. However, metagenome assemblies often generate enormously and yet incomplete protein databases, which undermines the identification of peptides and proteins in metaproteomics. Dr. Guo's group has developed a series of tools for identifying proteins from microbes. (1) Sipros Ensemble was designed for database searching using an ensemble approach. Benchmarking with soil and marine microbial communities demonstrated a higher number of peptide and protein identifications by Sipros Ensemble than popular proteomics search tools. Sipros Ensemble was computationally efficient and scalable on supercomputers. Sipros Ensemble was applied to microbial communities in P-deficient soils and P-rich soils, from which microbial communities were found to enhance the degradation of P-containing nucleic acids and phospholipids and the decomposition of labile carbon and nitrogen substrates. In contrast, microbial communities in the P-rich soils promoted the degradation of recalcitrant aromatic compounds, the transformation of nitrogenous compounds, and the assimilation of sulfur from taurine. (2) MetaLP was developed by Dr. Guo's group for inferring microbial proteins given a list of peptides using an integrative linear programming model. The protein inference task was formulated as an optimization problem that can be effectively solved using linear programming. Taxonomic abundance information, which was extracted from whole-genome sequencing or 16s rRNA gene sequencing, was incorporated as prior information in MetaLP. Benchmarking with mock, human gut, soil, and marine microbial communities demonstrated a higher number of protein identifications by MetaLP than the ProteinLP, PeptideProphet, DeepPep, and Sipros Ensemble frameworks. MetaLP substantially improved protein inference in complex metaproteome samples.
        <ol type="a">
          <li> Feng, S., Wang, H., Zhang, B., Sterzenbach, R., Pan, C., and Guo, X., 2022. MetaLP: An Integrative Linear Programming Method for Protein Inference in Metaproteomics. In submission to PLoS Computational Biology.</li>
          <li>Feng, S., Sterzenbach, R. and Guo, X., 2021. Deep learning for peptide identification from metaproteomics datasets. Journal of Proteomics, 247, p.104316. https://doi.org/10.1016/j.jprot.2021.104316</li>
          <li>Li, Z., Yao, Q., Guo, X., Crits-Christoph, A., Mayes, M.A., Lebeis, S.L., Banfield, J.F., Hurst, G.B., Hettich, R.L. and Pan, C., 2019. Genome-resolved proteomic stable isotope probing of soil microbial communities using 13CO2 and 13C-methanol. Frontiers in microbiology, 10, p.2706. https://doi.org/10.3389/fmicb.2019.02706</li>
          <li> Yao, Q., Li, Z., Song, Y., Wright, S.J., Guo, X., Tringe, S.G., Tfaily, M.M., Paša-Tolić, L., Hazen, T.C., Turner, B.L. and Mayes, M.A., 2018. Community proteogenomics reveals the systemic impact of phosphorus availability on microbial functions in tropical soil. Nature ecology & evolution, 2(3), p.499. https://doi:10.1038/s41559-017-0463-5 </li>
          <li> Guo, X., Li, Z., Yao, Q., Mueller, R.S., Eng, J.K., Tabb, D.L., Hervey IV, W.J. and Pan, C., 2018. Sipros ensemble improves database searching and filtering for complex metaproteomics. Bioinformatics, 34(5), pp.795-802. https://doi.org/10.1093/bioinformatics/btx601 </li>
        </ol>
        <li> <b> Detecting Gene-Gene Interactions in Genome-wide Case-Control Studies </b></li>
        Large genome-wide association studies (GWASs) have been considered to hold promise for unraveling complex relationships between genotype and phenotype. Dr. Guo’s group has developed many fast and powerful methods, such as JS-MA, TS, DCHE, DAM, MSCD, etc., for detecting genome-wide interactions related to multiple genes under different scenarios. Experimental results on simulated data demonstrate that these methods were more robust than some recently proposed methods. These methods have confirmed and discovered many novel high-order associations significantly rich in cases with different diseases, including Age-related Macular Degeneration, Rheumatoid Arthritis, and Type 1 Diabetes.
        <ol type="a">
          <li>Zhang, J., Guo, X., Gonzales, S., Yang, J. and Wang, X., 2020. TS: a powerful truncated test to detect novel disease associated genes using publicly available gWAS summary data. BMC bioinformatics, 21, pp.1-15. https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3511-0</li>
          <li>Guo, X., 2020. JS-MA: A Jensen-Shannon Divergence Based Method for Mapping Genome-Wide Associations on Multiple Diseases. Frontiers in genetics, 11, p.1251. https://doi.org/10.3389/fgene.2020.507038</li>
          <li>Wu, G., Guo, X. and Xu, B., 2020. BAM: A block-based Bayesian method for detecting genome-wide associations with multiple diseases. Tsinghua Science and Technology, 25(5), pp.678-689. https://doi.org/10.26599/TST.2019.9010064</li>
          <li>Zhang, J., Zhao, Z., Guo, X., Guo, B., & Wu, B., 2019. Powerful statistical method to detect disease-associated genes using publicly available genome-wide association studies summary data. Genetic epidemiology, 43(8), 941–951. https://doi.org/10.1002/gepi.22251 </li>
          <li>Ding, X. and Guo, X., 2018. A Survey of SNP Data Analysis. Big Data Mining and Analytics, 1(3), pp.173-190. https://doi.org/10.26599/BDMA.2018.9020015 </li>
          <li> Guo, X., Zhang, J., Cai, Z., Du, D.Z. and Pan, Y., 2016. Searching genome-wide multi-locus associations for multiple diseases based on Bayesian inference. IEEE/ACM transactions on computational biology and bioinformatics, 14(3), pp.600-610. DOI: 10.1109/TCBB.2016.2527648</li>
          <li> Ding, X., Wang, J., Zelikovsky, A., Guo, X., Xie, M. and Pan, Y., 2014. Searching high-order SNP combinations for complex diseases based on energy distribution difference. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 12(3), pp.695-704. doi&gt;10.1109/TCBB.2014.2363459</li>
          <li> Guo, X., Meng, Y., Yu, N. and Pan, Y., 2014. Cloud computing for detecting high-order genome-wide epistatic interaction via dynamic clustering. BMC bioinformatics, 15(1), p.102.         https://doi.org/10.1186/1471-2105-15-102</li>
        </ol>
        <li> <b> Metagenomics Sequence Assembling </b></li>
        In metagenomics, the population sequencing is the approach to recover the genomic sequences in a genetically diverse environment. Dr. Guo has developed a novel metagenomic sequence assembly framework for achieving a trade-off between the noise of the assembled contigs and the gain in the sequence length for better annotation for large-scale sequencing projects. The systematical comparison with other popular short read assembly programs on both synthetic and real metagenomic sequence datasets demonstrate that the novel framework not only partitions the sequence reads with extremely high accuracy, but also reconstructs more bases, generates higher quality assembled contigs, and yields higher assembly scores with a nearly linear speed-up.
        <ol type="a">
          <li>Guo, X., Yu, N., Ding, X., Wang, J. and Pan, Y., 2015. Dime: a novel framework for de novo metagenomic sequence assembly. Journal of Computational Biology, 22(2), pp.159-177.         https://doi.org/10.1089/cmb.2014.0251</li>
          <li>Guo, X., Yu, N., Li, B. and Pan, Y. (2016) Cloud Computing for Next-Generation Sequencing Data Analysis, in Computational Methods for Next Generation Sequencing Data Analysis (eds I. Mandoiu and A. Zelikovsky), John Wiley &amp; Sons, Inc., Hoboken, NJ, USA. doi: 10.1002/9781119272182.ch1</li>
          <li>Guo X, Ding X, Meng Y, Pan Y. Cloud computing for de novo metagenomic sequence assembly. In International Symposium on Bioinformatics Research and Applications 2013 May 20 (pp. 185-198). Springer, Berlin, Heidelberg.</li>
        </ol>
      </ul>
    </div>
    <div class="funded-project">
      <p></p>
    </div>
    <div class="services"> </div>
  </div>
  <div id="foot-placeholder"> </div>
</div>
</body>
</html>
