<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="application/xhtml+xml; charset=UTF-8" />
<link href="stylesheets/w3.css" type="text/css" rel="stylesheet" />
<link href="stylesheets/common.css" type="text/css" rel="stylesheet" />
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<title>NIH R15 project</title>
<script>
      $(function(){
	$("#header-placeholder").load("header.html");
	$("#foot-placeholder").load("foot.html");
}); </script>
</head>
<body>
<div class="main-container">
  <div id="header-placeholder"></div>
  <div class="DivIndent">
    <h4>NIH/NLM R15 Project</h4>
    <h2> A Computational Framework for Protein Identification and Quantification in Metaproteomics Using Data-Independent Acquisition</h2>
    <hr />
    <h3>Software</h3>
    <ul>
	  <li><b>CloudProteoAnalyzer</b></li>
	  <p>CloudProteoAnalyzer is a cloud-based parallel computing application to offer end-to-end proteomics data analysis software as a service</p>
	  <p>CloudProteoAnalyzer is available at <a href="https://sipros.unt.edu/">https://sipros-oscer.unt.edu/</a></p>
	  <p>The source code of CloudProteoAnalyzer is available at <a href="https://github.com/Biocomputing-Research-Group/CloudProteoAnalyzer">https://github.com/Biocomputing-Research-Group/CloudProteoAnalyzer</a> </p>
	  
	  <li><b>SEMQuant</b></li>
	  <p>SEMQuant is a protein quantification tool with match-between-runs in metaproteomics. It can convert the format of proteins identified by the database searching tool, i.e., Sipros, to the format accepted by the quantification tool, i.e., IonQuant</p>
	  <p>SEMQuant is available at <a href="https://github.com/Biocomputing-Research-Group/SEMQuant">https://github.com/Biocomputing-Research-Group/SEMQuant</a> </p>
      
	  <li><b>Transformer-DIA</b></li>
	  <p>Transformer-DIA is a transformer-based de novo peptide sequencing method for data-independent acquisition mass spectrometry.</p>
	  <p>Transformer-DIA is available at <a href="https://github.com/Biocomputing-Research-Group/Transformer-DIA">https://github.com/Biocomputing-Research-Group/Transformer-DIA</a> </p>
	  
	  <li><b>FineFDR</b></li>
	  <p>FineFDR is a fast and accurate fine-grained taxonomy-specific FDR assessment tool to control the FDR separately for peptides in different taxonomic ranks.</p>
	  <p>FineFDR is available at <a href="https://github.com/Biocomputing-Research-Group/FDR">https://github.com/Biocomputing-Research-Group/FDR</a> </p>

	  <li><b>WinnowNet</b></li>
	  <p>WinnowNet is a filtering algorithm with a well-calibrated re-scoring function for peptide identifications in metaproteomics.</p>
	  <p>WinnowNet is available at <a href="https://github.com/Biocomputing-Research-Group/WinnowNet">https://github.com/Biocomputing-Research-Group/WinnowNet</a> </p>
	  
	  <li><b>MS-feature</b></li>
	  <p>It is  a deep learning-based feature detection tool for feature detection in MS2 scans.</p>
	  <p>The source code of this tool is available at <a href="https://github.com/Biocomputing-Research-Group/MS-feature">https://github.com/Biocomputing-Research-Group/MS-feature</a> </p>
	  
	  <li><b>IDIA</b></li>
	  <p>IDIA is a tool to generate pseudo-spectra from data independent acquisition (DIA) data mass spectrometry-based proteomics data.</p>
	  <p>IDIA is available at <a href="https://github.com/Biocomputing-Research-Group/IDIA">https://github.com/Biocomputing-Research-Group/IDIA</a> </p>
      
	  <li><b>DeepFilter</b></li>
      <p>DeepFilter is a metaproteomics-filtering tool based on deep learning model. It is aimed at improving the improving peptide identifications of microbial communities from a collection of tandem mass spectra. </p>
      <p>DeepFilter is available at <a href="https://github.com/Biocomputing-Research-Group/DeepFilter">https://github.com/Biocomputing-Research-Group/DeepFilter</a></p>
      
	  <li><b>MetaLP</b></li>
      <p>MetaLP is a protein inference algorithm for shotgun proteomics analysis of microbial communities. Two key innovations in MetaLP were the integration of taxonomic abundances as prior information and the formulation of protein inference as a linear programming problem. It was optimized for metaproteomics to address the degenerate peptides and ‘one-hit wonders’ issues. The objective of MetaLP is to produce substantially higher numbers of protein identifications in complex metaproteomics datasets. </p>
      <p>MetaLP is available at <a href="https://github.com/Biocomputing-Research-Group/MetaLP">https://github.com/Biocomputing-Research-Group/MetaLP</a></p>
    </ul>
    <hr />
    <h3> Research Team </h3>
    <ul>
      <li>Dr. Xuan Guo (PI)</li>
      <li>Dr. Armin Milker (Co-I)</li>
      <li>Dr. Chongle Pan (Collaborator)</li>
      <li>Jiancheng Li (Graduate Research Assistant)</li>
      <li>Shichao Feng (Graduate Research Assistant)</li>
      <li>Ryan Sterzenbach (Undergraduate Research Assistant)</li>
      <li>Gloire Eunice Benga Ikambouayat (Undergraduate Research Assistant)</li>
      <li>Shengze Wang (Undergraduate Research Assistant)</li>
      <li>Kulthum Lakha(Undergraduate Research Assistant)</li>
      <li>Jonathon He (TAMS student)</li>
      <li>Oliva Liu(TAMS student)</li>
    </ul>
    <hr />
    <h3>Research Oppotunities</h3>
    <p> Undergraduate Research Fellowship is available for this project. This research program aims to develop novel computation tools for identifying and quantifying proteins of microbial mixtures. Undergraduate students are provided with the opportunities to learn mass spectrometry techniques and applied programming and practice with MS-based protein analysis. More details are at <a href="http://www.cse.unt.edu/~xuanguo/underfellowship.html">here</a>. 
  </div>
  <div id="foot-placeholder"> </div>
</div>
</body>
</html>
